service/internal/access/v2/validators.go

```diff
@@ -41,6 +42,28 @@ func validateGetDecision(entityRepresentation *entityresolutionV2.EntityRepresen
 	return nil
 }
 
+// validateGetDecisionRegisteredResource validates the input parameters for GetDecisionRegisteredResource:
+//   - registeredResourceValueFQN: must be a valid registered resource value FQN
+//   - action: must not be nil
+//   - resources: must not be nil and must contain at least one resource
+func validateGetDecisionRegisteredResource(registeredResourceValueFQN string, action *policy.Action, resources []*authzV2.Resource) error {
```

Comment:
Can we please add a TODO or follow up issue for a unit test of these behaviors so we can detect regressions if something changes logically?